test/abstract_syntax_tree.coffee

```diff
@@ -278,13 +291,13 @@ test "AST as expected for ComputedPropertyName node", ->
     implicit: yes
 
 test "AST as expected for StatementLiteral node", ->
-  testExpression 'break',
+  testStatement 'break',
```

Comment:
By being explicit about whether we're testing an "expression" (which is wrapped in an `ExpressionStatement`) or a statement, we're testing that the AST is correctly doing that wrapping or not for different node types